docs/MIGRATION.md

```
@@ -21,247 +21,240 @@ These files may contain **plaintext secrets**:
 
-| Type | Fields Migrated | Example ENV Name |
-|------|----------------|------------------|
-| `api_key` | `key` | `OPENCLAW_OPENAI_OPENAI_DEFAULT_KEY` |
-| `oauth` | `access`, `refresh` | `OPENCLAW_GOOGLE_GOOGLE_DEFAULT_ACCESS` |
-| `oauth` | `accessToken`, `refreshToken` | `OPENCLAW_CUSTOM_TEST_ACCESSTOKEN` |
+When OpenClaw needs a secret, it sends the exec provider ID to ClawVault:
 
-### Naming Convention
-
-Default ENV var names:
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)